subsys/testsuite/ztest/Kconfig

```diff
@@ -41,6 +41,13 @@ config ZTEST_MOCKING
 	  Enable mocking support for Ztest. This allows the test to set
 	  return values and expected parameters to functions.
 
+config ZTEST_TC_UTIL_USER_OVERRIDE
+       bool "Override tc_util.h"
+       depends on ZTEST
+       help
+               Enable overriding some defines in tc_util.h.
```

Comment:
Would be great to point any documentation which should helps to get know what should be provided as the tc_util_user_override.h. file content. Also on which exactly directory the file has to be located.

Reply:
^^ doc might be even a header template file, or a header in an example which shows this feature

Reply:
> 
> 
> Would be great to point any documentation which should helps to get know what should be provided as the tc_util_user_override.h. file content. Also on which exactly directory the file has to be located.

I updated the help section with more details. Should I change it further?

Reply:
> 
> 
> ^^ doc might be even a header template file, or a header in an example which shows this feature

I am working on a PR for fw-nrfconnect-nrf which will be using this feature, so at a later stage I could point to it.

Or I could just provide a template header. In that case do you have a suggestion on where to put it?

Reply:
I would add and extra simple sample to upstream which shows usage and has template you can point. Otherwise put it to ztest includes somewhere.